calculator.rb

```diff
@@ -0,0 +1,61 @@
+again = "y"
+
+while again == "y" do 
+
+    puts "Welcome to the Calculator!"
+    puts "Which operator would you like to use?"
+    list = ["add, +","substract, -","multiply *","divide /"]
+    list.length.times do |n|
+        puts "#{n+1}. #{list[n]}"
+    end 
+
+    user_operator = gets.chomp.downcase
+
+
+    until ["add","+","substract","-","multiply","*", "divide","/"].include?(user_operator)
+        puts "Enter an operator please!!"
+        user_operator = gets.chomp.downcase
+    end
+
+    nums = []
+
+    # function to ask for nums and store it in num array
+    def print_asking_num(nums)
+        puts "Enter Num 1"
```

Comment:
You've written almost exactly the same code here twice, to get the first number and the second number. Could you DRY that up by putting this logic in a method?